front/app/components/SeatInfo/index.tsx

```diff
@@ -13,52 +13,61 @@ import {
 // Hooks
 import useAppConfiguration from 'api/app_configuration/useAppConfiguration';
 import useSeats from 'api/seats/useSeats';
+import { TSeatType } from 'api/seats/types';
+import { TSeatNumber } from 'api/app_configuration/types';
 
 // Intl
 import messages from './messages';
-import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import { FormattedMessage, MessageDescriptor, useIntl } from 'utils/cl-intl';
 
 // Utils
 import { isNil } from 'utils/helperUtils';
 
-type SeatInfoType = {
-  seatType: 'collaborator' | 'admin';
+type Props = {
+  seatType: TSeatType;
 };
 
-const SeatInfo = ({ seatType }: SeatInfoType) => {
+const SeatInfo = ({ seatType }: Props) => {
   const { formatMessage } = useIntl();
   const { data: appConfiguration } = useAppConfiguration();
   const { data: seats } = useSeats();
-  const maximumAdmins =
-    appConfiguration?.data.attributes.settings.core.maximum_admins_number;
-  const maximumProjectCollaborators =
-    appConfiguration?.data.attributes.settings.core.maximum_moderators_number;
-  const maximumSeatNumber = {
-    admin: maximumAdmins,
-    collaborator: maximumProjectCollaborators,
-  }[seatType];
+
+  const maximumSeatNumbers: {
+    [key in TSeatType]: TSeatNumber;
+  } = {
+    admin:
+      appConfiguration?.data.attributes.settings.core.maximum_admins_number,
+    collaborator:
+      appConfiguration?.data.attributes.settings.core.maximum_moderators_number,
```

Comment:
Removed some vars because there were many in this file, and they don't have extra value here imo.